test/integration/data_bags/ros_buildfarm_repository_signing_keys/test-beta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "test-beta",
+  "default": true,
```

Comment:
I stuck the default tag on the "middle" key to make sure that it works when not the first or the last. The "default" key is the one that our ros-archive-keyring package should set `SignedBy=` for.